human_perception:
  human_tracker:
    # Parameters for human_track_publisher.cpp
    ros_parameters:
      det_obj_topic: "/zed2/zed_node/odom"
      human_track_topic: "/human/interpolated_history"
      odom_topic: "/zed2/zed_node/odom"
      pub_frame_id: "map"
      pub_frame_rate: 15.0
      max_history_length: 12
      delay_tolerance: 3.0
      max_interp_interval: 0.4